src/main/kotlin/no/nav/familie/ef/søknad/mapper/Språktekster.kt

```diff
@@ -28,7 +28,7 @@ enum class Språktekster(val norsk: String, val engelsk: String) {
     Sivilstatus("Sivilstatus", "Marital status"),
     Telefonnummer("Telefonnummer", "Telephone number"),
     Adresse("Adresse", "Address"),
-    OmSamboer("Om samboeren din", "About your cohabitant"),
+    OmSamboer("Om den tidligere samboeren din", "About your former cohabitant"),
```

Comment:
Denne brukes på 2 steder
Gjelder dette for begge tilfeller? 
Burde vi rename "OmSamboer" til `OmTidligereSamboer`?

Reply:
Ai - må sjekke! Takk :)

Reply:
Takk for tilbakemelding på denne @blommish . Det så riktig(ere) ut i noen tilfeller, men ble helt feil i andre. Overskriften ble brukt om både tidligere og nåværende samboere. Tror vi lar det stå som det er. :+1: